### PR TITLE
Two helper functions to sort questions and answers

### PR DESCRIPTION
**Pull request by austinchapin1**
@rvtaylor94 Just a couple helper functions to sort question data

### CODE REVIEW

File: src/components/Questions/sampleData.js [hunk 1]
```diff
@@ -0,0 +1,907 @@
...
+}
+
+
+// var testing = () => {
+//   sampleData.results.forEach(question => {
+//     console.log(question.question_body)
+//   })
+// }
+
+debugger;
```
Comment (rvtaylor94) [inline src/components/Questions/sampleData.js:882]: I'm not sure if lines 882 - 905 need to be in the main code. Looks like you were using them for testing. Could you elaborate?

Comment (austinchapin1) [reply src/components/Questions/sampleData.js:882]: No you're right, got trigger happy. I'll see if I can adjust in that branch and push it up, tbd.

Comment (austinchapin1) [reply src/components/Questions/sampleData.js]: @rvtaylor94 My brain is done for the night lol. Should be good now.

**Review by rvtaylor94 — APPROVED** [review]
Looks good!

**Pull request merged**